module2/homework/grayscaleImages/test.cpp

```diff
@@ -0,0 +1,190 @@
+#include <algorithm>
+#include <array>
+#include <utility>  // for std::pair<>
+#include <vector>
```

Comment:
You could remove array and vector include here since they are in the header.

Reply:
+1